scheduler/src/cook/components.clj

```diff
@@ -61,15 +62,28 @@
                 [:headers "Cache-control"]
                 "max-age=0"))))
 
+(def rate-limiters
+  {:create-job-rate-limiter
+   (fnk [[:settings [:rate-limit expire-minutes job-creation]]]
+     (if job-creation
+       (let [{:keys [bucket-size is-enforcing rate-per-minute]} job-creation]
+         (when-not (or (true? is-enforcing) (false? is-enforcing))
```

Comment:
We can use `boolean?` here instead of checking `true?` and `false?`.

Reply:
`boolean?` is not available in clojure 1.8.